docs/self-managed/zeebe-deployment/operations/cluster-scaling.md

```diff
@@ -0,0 +1,182 @@
+---
+id: cluster-scaling
+title: "Cluster Scaling"
+description: "How to scale a Zeebe cluster"
+---
+
+:::note
+This is an experimental feature.
+:::
+
+Zeebe allows scaling an existing cluster. When new brokers are added, the partitions will be redistributed to the new brokers thus distributing some load. The cluster can also be scaled down to remove extra brokers.
+
+Zeebe provides a REST API to manage the cluster scaling. The cluster management API is a custom endpoint available via [Spring Boot Actuator](https://docs.spring.io/spring-boot/docs/3.1.x/reference/htmlsingle/#actuator.endpoints). This is accessible via the management port of the gateway.
+
+## Configuration
+
+- This is an experimental feature. To use the feature set `ZEEBE_BROKER_EXPERIMENTAL_FEATURES_ENABLEDYNAMICCLUSTERTOPOLOGY` to `true`.
```

Comment:
:thought_balloon: I think we should explain what happens when users disable the feature after changing the cluster topology.

Reply:
If we want to go the extra mile, we could also explain how to do "rollbacks" by deleting the topology files.

Reply:
> If we want to go the extra mile, we could also explain how to do "rollbacks" by deleting the topology files.

This would be risky. Let's add it in the next iteration.

Reply:
> Also, when this is enabled, fixed partition distribution is not respected!

:+1: Good point. I will add that.